readme.md

```diff
@@ -2,3 +2,9 @@ This script aims to compare three different numerical methods to solve pendulum
 
 # Results
 ![pendulum](./Pic/pendulum.png)
+
+# comparision
+
+![Euler](./Pic/Euler.gif)
+![BackwardEuler](./Pic/BackwardEuler.gif)
+![SemiEuler](./Pic/SemiEuler.gif)
```

Comment:
Awesome work! And next we can implement other numerical  integration methods, such as verlet integration and R-K integration. Besides, adding some explanation to different methods is also meaningful. And maybe this can be a simple, straightforward tutorial to numerical integration in the future.